keywords/LiteServNetMono.py

```diff
@@ -26,7 +26,11 @@ def download(self):
         """
 
         # Skip download if packages is already downloaded
-        expected_binary = "{}/couchbase-lite-net-mono-{}-liteserv/net45/LiteServ.exe".format(BINARY_DIR, self.version_build)
+        if self.version_build.startswith("1.2") or self.version_build.startswith("1.3") or self.version_build.startswith("1.4.0"):
```

Comment:
any reason not to use `self.version_build.startswith("1.4")` to be consistent with other comparisons?